search/tests/utils.py

```diff
@@ -73,7 +76,8 @@ def search(self,
 class ErroringIndexEngine(MockSearchEngine):
     """ Override to generate search engine error to test """
 
-    def index(self, sources, **kwargs):  # pylint: disable=unused-argument, arguments-differ
+    @doc_type_warning_decorator
+    def index(self, sources, doc_type=None, **kwargs):  # pylint: disable=unused-argument, arguments-differ
```

Comment:
Please confirm that you rolled back to the old interface and doc_type in the proper place and was non-required before too.

Reply:
I return doc_type in all methods that can be called with this parameter, I think)